### PR TITLE
export component prop-types

### PR DESCRIPTION
**Pull request by adnasa**
#### Summary

- export component prop-types

ref: https://github.com/commercetools/merchant-center-application-kit/pull/2193/files

### CODE REVIEW

File: packages/components/inputs/text-input/src/text-input.tsx [hunk 1]
```diff
@@ -61,7 +61,7 @@ type TTextInputProps = {
   /**
    * Horizontal size limit of the input fields.
    */
-  horizontalConstraint:
+  horizontalConstraint?:
```
Comment (mohib0306) [inline packages/components/inputs/text-input/src/text-input.tsx:64]: Shouldn't the readme be regenerated?

**Review by mohib0306 — APPROVED** [review]
LGTM

**Review by ahmehri — APPROVED** [review]

File: packages/components/inputs/text-input/src/text-input.tsx (hunk 1 above)
Comment (adnasa) [reply packages/components/inputs/text-input/src/text-input.tsx:64]: thanks!

Comment (adnasa) [reply packages/components/inputs/text-input/src/text-input.tsx:64]: https://github.com/commercetools/ui-kit/pull/1892/commits/fe2eff259d71680c070f6507dfccfd7d8c3ebc18

**Review by mohib0306 — APPROVED** [review]

**Pull request merged**